packages/contracts-bedrock/semver-lock.json

```diff
@@ -68,8 +68,8 @@
     "sourceCodeHash": "0x3a725791a0f5ed84dc46dcdae26f6170a759b2fe3dc360d704356d088b76cfd6"
   },
   "src/L2/CrossL2Inbox.sol": {
```

Comment:
You can see in the `semver-lock.json` that both the `CrossL2Inbox` and the `L2ToL2CrossDomainMessenger` had their sourcecode + bytecode modified, so both should receive a version bump. Since they are prerelease, its good to bump the right most integer